spec/student_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'all_methods'
+
+describe Student do
+  context 'When testing the Student class, it' do
+    physics = Classroom.new('Physics')
+    parent_permission = true
+    tokyo = Student.new(nil, physics, 16, 'Tokyo', parent_permission)
+
+    it 'should create a new student' do
+      expect(tokyo).to be_a(Student)
```

Comment:
- [ ] You did well testing some of your methods inside the Student class, But as part of the main requirements for this milestone, we need to have an 'it' block for each class method (at least). Please add a test case for your `to_json, play_hooky, and add_classroom` methods inside the Student class.

Reply:
- Well resolved 👍🏽